.github/workflows/data/local/main.tf

```
@@ -0,0 +1,5 @@
+resource "null_resource" "null" {
+  triggers = {
+    value = timestamp()
+  }
+}
```

```suggestion
resource "terraform_data" "replacement" {
  triggers_replace = timestamp()
}
```

The `terraform_data` resource type is the native `null_resource` [replacement](https://developer.hashicorp.com/terraform/language/resources/terraform-data#example-usage-null_resource-replacement).

---

Good catch! Thank you! For context, this code was copied over from setup-terraform 🙂

---

Implemented